sql/core/src/test/resources/sql-tests/results/describe-table-after-alter-table.sql.out

```diff
@@ -25,6 +25,7 @@ Database            	default
 Table               	table_with_comment  	                    
 Created [not included in comparison]
 Last Access [not included in comparison]
+Create Version [not included in comparison]
```

Comment:
I manually verified all these version values are right. To avoid the unnecessary test result updates, hide the values.